compiler/AST/UseStmt.cpp

```diff
@@ -157,9 +157,8 @@ bool UseStmt::isARename(const char* name) const {
   return renamed.count(name) == 1;
 }
 
-// Specifically for when the module being used is renamed
 bool UseStmt::isARename() const {
-  return modRename[0] != '\0';
+  return VisibilityStmt::isARename();
```

Comment:
Why not remove this method entirely from UseStmt? Since it inherits from VisibilityStmt then isARename() will still be called. Ditto getRename and the ImportStmt versions. And checkIfModuleNameMatches.

Reply:
This one and the getRename method needed to be explicitly defined on UseStmt due to the presence of versions with arguments (for when the only list specifies renames) - the compiler complained about there being a getRename with an argument on UseStmt and that it wouldn't  overload the other version.  I'll look into if checkIfModuleNameMatches can be removed, though

Reply:
In cases such as this, I don't think it's "worth it" to have the two methods have the same name. Just make the one taking an argument have a different name, like `getRenameFor` or something.

Reply:
Depending on how the import discussion finishes, though, we might end up moving those methods to VisibilityStmt as well so I'm not super concerned.